### ISSUE TITLE
IosSafariOptions.presentationStyle affecting only dismiss animation

### ISSUE THREAD

**Issue by rokgregoric** [pichillilorenzo/flutter_inappwebview#305]
## Environment

**Flutter version:** 1.12.13+hotfix.8
**Plugin version:**  2.1.0+1
**Android version:**    
**iOS version:**
**Xcode version:**  
**Device information:**    

## Description

**Expected behavior:** 
On iOS the presentationStyle of SafariViewController should be the same for presenting/dismissing the controller.

**Current behavior:** 
IosSafariOptions.presentationStyle passed in the initializer have no effect on the presentation but they affect the dismissal.

## Steps to reproduce

```
ChromeSafariBrowser().open(
  url: 'https://google.com',
  options: ChromeSafariBrowserClassOptions()
);
```
opens/closes this way: [https://www.dropbox.com/s/l37aob1amr9by47/no-options.mov?dl=0](https://www.dropbox.com/s/l37aob1amr9by47/no-options.mov?dl=0)

```
ChromeSafariBrowser().open(
  url: 'https://google.com',
  options: ChromeSafariBrowserClassOptions(
    iosSafariOptions: IosSafariOptions(
      presentationStyle: IosWebViewOptionsPresentationStyle.OVER_FULL_SCREEN
    )
  )
);
```
only affects the dismissal: [https://www.dropbox.com/s/vhl0ivgmvz55xli/with-options.mov?dl=0](https://www.dropbox.com/s/vhl0ivgmvz55xli/with-options.mov?dl=0)

**Comment by ArkeshGKalathiya** [comment]
Same Issue here! Have you found any fix?

**Comment by ayush221b** [comment]
I am facing this issue as well, I tried going through the code but couldn't find anything missing @pichillilorenzo can you please look into it once?

**Comment by pichillilorenzo** [comment]
Fixed with new released version `3.1.0` right now!
